wsf_scraping/spiders/gov_spider.py

```diff
@@ -10,20 +11,47 @@ class GovSpider(BaseSpider):
         'JOBDIR': 'crawls/gov_uk'
     }
 
+    def __init__(self, *args, **kwargs):
+        year_before = kwargs.get('year_before', False)
```

Comment:
oops, sorry, one more thing - is it worth clarifying in the whether `year_before` etc are inclusive? i.e. if I run it with args 2013 - 2015, is that all of 2013, 2014, 2015? (yes, on reading the code)

Reply:
I'll add some comments 👍